staging/src/k8s.io/apiserver/pkg/util/openstack/flavors.go

```diff
@@ -21,6 +21,8 @@ import "fmt"
 var flavors = map[string]FlavorType{}
 var flavorList = []*FlavorType{}
 
+var ERROR_FLAVOR_NOT_FOUND = fmt.Errorf("flavor not found")
```

Comment:
const?

Reply:
it cant be. the "fmt.Errorf("flavor not found")" is not a const.